internal/handlers/pg/pgdb/query_documents.go

```diff
@@ -141,12 +148,21 @@ func iterateFetch(ctx context.Context, fetched chan FetchedDocs, rows pgx.Rows)
 				return writeFetched(ctx, fetched, FetchedDocs{Err: lazyerrors.Error(err)})
 			}
 
-			doc, err := fjson.Unmarshal(b)
-			if err != nil {
-				return writeFetched(ctx, fetched, FetchedDocs{Err: lazyerrors.Error(err)})
+			if explain {
+				var plans []map[string]any
+				if err := json.Unmarshal(b, &plans); err != nil {
```

Comment:
Let's move `json.Unmarshal` to those new functions too

Reply:
> Let's move `json.Unmarshal` to those new functions too

extracted to function.